go-ussd_registration.js

```diff
@@ -1377,8 +1378,34 @@ go.app = function() {
                     "Please dial back into {{channel}} to complete the Hello MAMA registration"
                     );
             });
-        };
 
+            self.env = self.im.config.env;
+            self.metric_prefix = [self.env, self.im.config.name].join('.').replace(/-/g, '_');
+            self.store_name = [self.env, self.im.config.name].join('.');
+
+            mh = new MetricsHelper(self.im);
+            mh
+                .add.trigger({
+                    action: 'enter',
+                    state: 'state_msg_receiver'
+                }, {
+                    total_state_actions: [self.metric_prefix, "registrations_started"].join('.'),
+                })
+                .add.trigger({
+                    action: 'enter',
+                    state: 'state_end_voice'
+                }, {
+                    total_state_actions: [self.metric_prefix, "registrations_completed"].join('.'),
+                })
+                .add.trigger({
+                    action: 'enter',
+                    state: 'state_end_sms'
+                }, {
+                    total_state_actions: [self.metric_prefix, "registrations_completed"].join('.'),
+                })
```

Comment:
Yeah, saw the first thing that would work in the documentation and went with it, let me change it, Please wait with the review on the voice one, I'll do the same there.